File: Assignment23/C/Program5.c
// PS - Accept string from user and count number of white spaces in it

#include <stdio.h>

int CountWhiteSpace(char *str)
{
    int iCount = 0;
    while (*str != '\0')
    {
        if (*str == ' ')
        {
            iCount++;
        }
        str++;
    }
    return iCount;
}

int main()
{
    char Arr[20];
    int iRet = 0;

    printf("Enter string \n");
    scanf("%[^'\n']s", Arr);

    iRet = CountWhiteSpace(Arr);

    printf("Count of white spaces are %d \n", iRet);

    return 0;
}

File: Assignment36/Java/Program5.java
 // PS - Write a java program which accept number from user and return product of all odd numbers

import java.util.*;

class ArrayX
{
    public int ProductX(int Arr[])
    {
        int Mult = 1;
        for(int iCnt = 0; iCnt < Arr.length; iCnt++)
        {
            if((Arr[iCnt] % 2) != 0)
            {
                Mult = Mult * Arr[iCnt];
            }
        }
        return Mult;
    }
}

class Program5
{
    public static void main(String arg[])
    {
        Scanner sobj = new Scanner(System.in);

        System.out.println("Enter number of elements");
        int iSize = sobj.nextInt();

        int Arr[] = new int[iSize];

        System.out.println("Enter elements of array");
        for(int iCnt = 0; iCnt < iSize; iCnt++)
        {
            Arr[iCnt] = sobj.nextInt();
        }

        ArrayX aobj = new ArrayX();
        int iRet = aobj.ProductX(Arr);

        System.out.println("Product of all odd numbers are : "+iRet);
    }
}